stripes-web/src/main/java/org/stripesframework/web/controller/UrlBindingFactory.java

```diff
@@ -79,6 +79,33 @@ public static UrlBinding parseUrlBinding( Class<? extends ActionBean> beanType )
       }
    }
 
+   /**
+    * Look for a additional binding patterns for the given {@link ActionBean} class, specified by the
+    * {@link org.stripesframework.web.action.UrlBinding} annotation.
+    *
+    * @param beanType The {@link ActionBean} type whose binding is to be parsed
+    * @return A list of {@link UrlBinding}
+    * @throws ParseException If the pattern cannot be parsed
+    */
+   public static List<UrlBinding> parseAdditionalUrlBindings(Class<? extends ActionBean> beanType ) {
+      // check that class is annotated
+      org.stripesframework.web.action.UrlBinding annotation = beanType.getAnnotation(org.stripesframework.web.action.UrlBinding.class);

```

Comment:
We "parse" this multiple times now, should we integrate this into `parseUrlBinding` now that we have everything in the same annotation?

Reply:
fixed